code/__DEFINES/rust_g.dm

```diff
@@ -38,12 +38,7 @@
 #define RUST_G (__rust_g || __detect_rust_g())
 #endif
 
-// Handle 515 call() -> call_ext() changes
-#if DM_VERSION >= 515
 #define RUSTG_CALL call_ext
-#else
-#define RUSTG_CALL call
-#endif
```

Comment:
Don't touch this, rust_g as a dependency supports pre-515 and this will be overridden

Reply:
o7